modules/karpenter/README.md

```diff
@@ -20,19 +21,19 @@ module "eks" {
 }
 
 module "karpenter" {
-  source = "terraform-aws-modules/eks/aws//modules/karpenter"
+  source = "../../modules/karpenter"
 
   cluster_name = module.eks.cluster_name
 
-  # Attach additional IAM policies to the Karpenter node IAM role
+  enable_pod_identity             = true
+  create_pod_identity_association = true
+
+  # Used to attach additional IAM policies to the Karpenter node IAM role
   node_iam_role_additional_policies = {
     AmazonSSMManagedInstanceCore = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore"
   }
 
-  tags = {
-    Environment = "dev"
-    Terraform   = "true"
-  }
+  tags = local.tags
```

Comment:
revert this as well - we're not showing how its coded in the examples, just showing a static example